xml/System.Net.Quic/QuicConnectionOptions.xml

```diff
@@ -118,8 +129,8 @@
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets the limit on the number of unidirectional streams the remote peer connection can create on an open connection.</summary>
```

Comment:
The wording suggests that the limit is for the total number of streams opened, but in reality it limits number of streams concurrently opened.

```suggestion
        <summary>Gets or sets the maximum number of concurrent unidirectional streams the remote peer connection can create on an open connection.</summary>
```

Reply:
```suggestion
        <summary>Gets or sets the maximum number of concurrent unidirectional streams that the remote peer connection can create on an open connection.</summary>
```